core/src/main/java/hudson/model/AbstractProject.java

```diff
@@ -210,6 +211,11 @@
     protected volatile boolean disabled;
 
     /**
+     * When the job is disabled.
+     */
+    protected long disabledSince;
```

Comment:
imho should be `Long` for not disabled project should have null instead of some long value.